test/integration/restart_test.rb

```diff
@@ -0,0 +1,37 @@
+require 'test_helper'
+require 'kubernetes-deploy/restart_task'
+
+class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
+  def test_restart
+    deploy_fixtures("basic", subset: ["configmap-data.yml", "web.yml.erb"])
```

Comment:
Because `web` from the fixtures is referring to config map values. That's how it works in all tests